crates/storage/db/src/tables/mod.rs

```diff
@@ -143,8 +143,8 @@ table!(
 );
 
 table!(
-    /// Stores the transaction body from non canonical transactions.
-    ( NonCanonicalTransactions ) BlockNumHashTxNumber | TransactionSigned
+    /// Stores full block that can be potentially included to canonical chain.
+    ( PendingBlocks ) BlockHash | Block
 );
```

Comment:
oh so we're still using the database to store blocks in the tree, makes sense

Reply:
It is for crash recovery, most run's are in  daemons that would restart the node if it crashes, so saving pending blocks can help use recover for that usecase.